src/Uno.UI.Tests/Windows_UI_XAML_Controls/UserControlTests/UserControl_WriteOnlyProperty_UserControl.xaml.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI.Tests.Windows_UI_XAML_Controls.UserControlTests
+{
+	public sealed partial class UserControl_WriteOnlyProperty_UserControl : UserControl
+	{
+		public string Text
```

Comment:
Codacy found an issue: [Provide a getter for 'Text' or replace the property with a 'SetText' method.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6347985)